Fix the sample and added a readme for the sample

JurrianFahner opened the pull request:
I tried the sample and found out that it didn't work out of the box. After a small change in the version of the pom.xml it seemed working correctly. 

To help other users to get started quickly, I've added a readme to the sample. 

If I need to improve this PR, please let me know...

JurrianFahner committed:
Downgrade dependency version, to make sample working

JurrianFahner committed:
Added readme for the sample

JurrianFahner committed:
Small text change

janfockaert commented:
Thanks for the contribution!

The sample was configured to use the latest development version 1.0.3-alpha. The development repository was not listed in the pom.xml which causes build failure. So we can list the development repository or use the latest release version. Can you add the development repository and see if this fixes the problem:

```xml
<repository>
    <id>bintray-merapar-maven</id>
    <name>bintray</name>
    <url>http://dl.bintray.com/merapar/maven</url>
</repository>
```

The sample readme gives a good overview of the possibilities. An improvement point is to list [GraphiQL](https://github.com/graphql/graphiql) application for easy playing with the API.

JurrianFahner commented:
Hi Jan,

Thanks for your feedback!

I'll try to fix the PR later today.

Regards,

Jurrian

Op ma 7 aug. 2017 09:26 schreef Jan Fockaert <notifications@github.com>:

> Thanks for the contribution!
>
> The sample was configured to use the latest development version
> 1.0.3-alpha. The development repository was not listed in the pom.xml which
> causes build failure. So we can list the development repository or use the
> latest release version. Can you add the development repository and see if
> this fixes the problem:
>
> <repository>
>     <id>bintray-merapar-maven</id>
>     <name>bintray</name>
>     <url>http://dl.bintray.com/merapar/maven</url>
> </repository>
>
> The sample readme gives a good overview of the possibilities. An
> improvement point is to list GraphiQL
> <https://github.com/graphql/graphiql> application for easy playing with
> the API.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/merapar/graphql-spring-boot-starter/pull/24#issuecomment-320589298>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AL037InnxV7WDyTQce8gXhO_iKPU-F_wks5sVrwRgaJpZM4OuvHH>
> .
>

JurrianFahner committed:
PR feedback implemented

- added development repository
- set the version to latest dev version
- added graphiql as a graphql client

janfockaert commented:
Thanks!
